HWs/Test3/task3/main.c

```diff
@@ -0,0 +1,72 @@
+#include <stdio.h>
+#include <stdbool.h>
+
+#define maximumSize 256
+
+bool isCorrectString(char *string)
+{
+    int state = 0;
+    char currentCharacter = -1;
+    int counter = 0;
+    bool isContinue = true;
+    while (isContinue && currentCharacter != '\0')
+    {
+        currentCharacter = string[counter++];
+        switch (state)
+        {
+        case 0:
+            if ((currentCharacter >= 'A' && currentCharacter <= 'Z') || (currentCharacter >= 'a' && currentCharacter <= 'z'))
+            {
+                state = 1;
+                break;
+            }
+            return false;
+        case 1:
+            if ((currentCharacter >= 'A' && currentCharacter <= 'Z') || (currentCharacter >= 'a' && currentCharacter <= 'z') ||
+                (currentCharacter >= '0' && currentCharacter <= '9') || (currentCharacter == '_'))
+            {
+                break;
+            }
+            isContinue = false;
```

Comment:
```suggestion
            return currentCharacter == '\0';
```
Лишняя переменная isContinue. 
(-0.25)